requirements.txt

```diff
@@ -5,3 +5,4 @@ netdiff>=0.6.0,<0.7
 openwisp-utils>=0.2,<0.3
 jsonfield
 six
+django-sortedm2m==2.0
```

Comment:
why are you adding sortedm2m? I think we can avoid this, I would use a standard django many2many field

Reply:
The standard django many2many field does not keep the order of addresses. So, the netjson id is not what your enter as the first address rather it is by alphabetical first item.